app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,52 @@
+<div class="navbar navbar-default">
+  <div class="navbar-inner">
+    <div class="container-fluid">
+      <div class="navbar-header">
+        <button type="button" class="navbar-toggle collapsed" data-target="#nav-collapse" data-toggle="collapse">
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </button>
+        <a><%= link_to image_tag('logo.png'), root_path, class: 'navbar-brand' %></a>
+      </div>
+      <div class="collapse navbar-collapse" id="nav-collapse">
+        <ul class="nav navbar-nav">
+          <li><%= link_to t('home'), root_path %></li>
+          <li><%= link_to Project.model_name.human, projects_path %></li>
```

Comment:
We could make use of the opportunity and make these link names plural, which makes it more obvious that there is a list of them to browse.